EncryptionTests/Encryption Tests/DatastoreEncryptionTests.m

```diff
@@ -192,4 +193,15 @@ - (void)testInitReturnsNilIfEncryptionKeyProviderDoesNotReturnTheKeyUsedToOpenTh
                  @"DB can not be opened with a wrong key so the datastore can not be initialised");
 }
 
+- (void)testTextSearchIsAvailable
+{
+    CDTHelperFixedKeyProvider *provider = [CDTHelperFixedKeyProvider provider];
+    
+    CDTDatastore *datastore = [self.factory datastoreNamed:@"test_copyprovider"
+                                 withEncryptionKeyProvider:provider
+                                                     error:nil];
+    
+    XCTAssertTrue([datastore isTextSearchEnabled], @"It should be activated");
```

Comment:
@gadamc: @alfinkel: I also copied this test from [here](https://github.com/cloudant/CDTDatastore/blob/master/Tests/Tests/CDTDatastoreQueryTests.m#L130)